app/src/main/java/com/greenmonkeys/handmade/payoutdenominations/USDollarWallet.java

```diff
@@ -0,0 +1,158 @@
+            runningBalance -= 50.0;
+        }
+        while (runningBalance >= 20.0) {
+            this.twenty++;
+            runningBalance -= 20.0;
+        }
+        while (runningBalance >= 10.0) {
+            this.ten++;
+            runningBalance -= 10.0;
+        }
+        while (runningBalance >= 5.0) {
+            this.five++;
+            runningBalance -= 5.0;
+        }
+        while (runningBalance >= 1.0) {
+            this.one++;
+            runningBalance -= 1.0;
+        }
+        while (runningBalance >= 0.25) {
+            this.quarter++;
+            runningBalance -= 0.25;
+        }
+        while (runningBalance >= 0.1) {
+            this.dime++;
+            runningBalance -= 0.1;
+        }
+        while (runningBalance >= 0.05) {
+            this.nickel++;
+            runningBalance -= 0.05;
+        }
+        while (runningBalance >= 0.01) {
+            this.penny++;
+            runningBalance -= 0.01;
+        }
+
+    }
+
+    public double total() {
+        return (hundred * 100.0) +
+                (fifty * 50.0) +
+                (twenty * 20.0) +
+                (ten * 10.0) +
+                (five * 5.0) +
+                (one * 1.0) +
+                (quarter * 0.25) +
+                (dime * 0.1) +
+                (nickel * 0.05) +
+                (penny * 0.01);
+    }
+
+    public int getHundred() {
+        return hundred;
+    }
+
+    public int getFifty() {
+        return fifty;
+    }
+
+    public int getTwenty() {
+        return twenty;
+    }
+
+    public int getTen() {
+        return ten;
+    }
+
+    public int getFive() {
+        return five;
+    }
+
+    public int getOne() {
+        return one;
+    }
+
+    public int getQuarter() {
+        return quarter;
+    }
+
+    public int getDime() {
+        return dime;
+    }
+
+    public int getNickel() {
+        return nickel;
+    }
+
+    public int getPenny() {
+        return penny;
+    }
+
+    public static USDollarWallet addWalletsInArray(USDollarWallet[] wallets) {
+        USDollarWallet result = new USDollarWallet(0.0);
+        for (USDollarWallet wallet : wallets) {
+            USDollarWallet previous = result;
+            result = add(result, wallet);
+        }
+        return result;
+    }
+
+    public static USDollarWallet add(USDollarWallet first, USDollarWallet second) {
+        return new USDollarWallet (
+                first.getHundred() + second.getHundred(),
+                first.getFifty() + second.getFifty(),
+                first.getTwenty() + second.getTwenty(),
+                first.getTen() + second.getTen(),
+                first.getFive() + second.getFive(),
+                first.getOne() + second.getOne(),
+                first.getQuarter() + second.getQuarter(),
+                first.getDime() + second.getDime(),
+                first.getNickel() + second.getNickel(),
+                first.getPenny() + second.getPenny()
+        );
+    }
+
```

Comment:
The adding functionality should be moved to a Wallet Manager class that performs the adding for the different wallets that our Artisans have. We will be able to use this new Wallet Manager class for the CGA as they are managing the Artisans.